scipp-imaging-2D.ipynb

```diff
@@ -196,246 +339,780 @@
     "\n",
     "    return frames[0]\n",
     "\n",
-    "def zero_special_values(values):\n",
-    "    np.nan_to_num(values, copy=False)\n",
-    "    values[values == sys.float_info.max] = 0"
+    "\n",
+    "# Data identical up to this point\n",
+    "def stitch_data(variable_to_stitch):\n",
+    "    print(f\"Stitching: {variable_to_stitch}\")\n",
+    "    stitched[variable_to_stitch] = stitch(ds[variable_to_stitch],\n",
+    "                                          frame_parameters=frame_parameters,\n",
+    "                                          frame_shifts=frame_shifts,\n",
+    "                                          rebin_parameters=rebin_parameters)\n",
+    "\n",
+    "# Comparing [\"sample\"][\"spectrum\", 37000] data stops matching, as Mantid\n",
+    "# and skip appear to handle to partial bin at the end differently. This results\n",
+    "# in different counts making data comparison difficult\n",
+    "for var_name in ds.keys():\n",
+    "    stitch_data(var_name)"
```

Comment:
Doesn't this work if we just call `stitch` for the whole dataset? Or is`rebin` for `Dataset` not supported yet?